### PR TITLE
feature: added wishlist

### PR DESCRIPTION
**Pull request by omgaikwad**
feature: added wishlist
Preview: https://deploy-preview-3--practical-feynman-c4afb1.netlify.app/pages/wishlist/wishlist.html

### CODE REVIEW

File: pages/wishlist/wishlist.html [hunk 1]
```diff
@@ -0,0 +1,205 @@
...
+      rel="stylesheet"
+    />
+    <link rel="stylesheet" href="../../css/main.css" />
+    <link rel="stylesheet" href="./wishlist.css" />
+    <title>3AM TALES</title>
+  </head>
+  <body>
+    <nav class="navbar" id="main-navbar">
+      <div class="left-navbar">
+        <!-- <img src="./assets/logo/3am tales.png" alt="" class="logo-img" /> -->
```
Comment (t007rushi) [inline pages/wishlist/wishlist.html:29]: i think you forgot to remove comments

**Review by t007rushi — APPROVED** [review]

File: pages/wishlist/wishlist.html [hunk 2]
```diff
@@ -0,0 +1,205 @@
...
+        <div class="logo-title">
+          <a href="" id="logo-title-link">3AM TALES</a>
```
Comment (shivampandey0) [inline pages/wishlist/wishlist.html:31]: empty `href` It's a logo link, so maybe it should go to `index.html`.

**Review by shivampandey0 — COMMENTED** [review]
Looks good with comments to identify the code block.
Added a suggestion

File: pages/wishlist/wishlist.html (hunk 1 above)
Comment (omgaikwad) [reply pages/wishlist/wishlist.html:29]: I will remove it.

File: pages/wishlist/wishlist.html (hunk 2 above)
Comment (omgaikwad) [reply pages/wishlist/wishlist.html:31]: Sure, I will add the link.

**Pull request merged**